### PR TITLE
WIP architecture: Add SOTA chapters

### PR DESCRIPTION
**Pull request by GordanM**
This commit adds chapters for the specific SOTA client used in the
PELUX Platform Baseline. Additional chapters about the generic
research, requirements and use cases is added through the SWF
blueprint.

Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>

### CODE REVIEW

File: docs/chapters/architecture/index.rst [hunk 1]
```diff
@@ -7,5 +7,6 @@ The following chapters describes the architecture of the PELUX platform.
 
    ../../swf-blueprint/docs/articles/architecture/arch-intro.rst
    ../../swf-blueprint/docs/articles/architecture/arch-concepts.rst
-   ../../swf-blueprint/docs/articles/architecture/arch-template.rst
+   ../../swf-blueprint/docs/articles/architecture//arch-vertical-configurations.rst
```
Comment (tobsan) [inline docs/chapters/architecture/index.rst]: extra / here

**Review by tobsan — CHANGES_REQUESTED** [review]
I suppose you want to bump the blueprint once that PR has been merged also, but looks good overall imho.

**Comment by GordanM** [comment]
Depends on: https://github.com/Pelagicore/software-factory-blueprint/pull/70

**Comment by GordanM** [comment]
@tobsan  I have updated the submodule.

**Review by tobsan — APPROVED** [review]

**Pull request merged**